DataConnectors/Morphisec.json

```diff
@@ -0,0 +1,106 @@
+{
+    "id": "Morphisec",
+    "title": "Morphisec UTPP",
+    "publisher": "Morphisec",
+    "descriptionMarkdown": "Integrate vital insights from your security products with the Morphisec Data Connector for Azure Sentinel and expand your analytical capabilities with search and correlation, threat intelligence, and customized alerts. Morphisec's Data Connector provides visibility into today's most advanced threats including sophisticated fileless attacks, in-memory exploits and zero days. With a single, cross-product view, you can make real-time, data-backed decisions to protect your most important assets",
+    "additionalRequirementBanner": "These queries and workbooks are dependent on Kusto functions based on Kusto to work as expected. Follow the steps to use the Kusto functions alias \"Morphisec\" \nin queries and workbooks. [Follow steps to get this Kusto function.](https://aka.ms/MorphisecFunction)",
```

Comment:
Please place this kusto link in the json "https://aka.ms/sentinel-Morphiesc-parser"

Reply:
fixed links to parser